codemeta.json

```
@@ -51,5 +32,446 @@
       "@type": "Organization",
       "name": "Smithsonian Institution"
     }
-  ]
+    ],
+    "keywords": [
+        "plasma physics",
+        "plasma",
+        "science"
+    ],
+    "programmingLanguage": [
+        "Python 3.6",
+        "Python 3.7",
+        "Python 3.8",
+        "Python 3.9"
+    ],
+    "operatingSystem": [
+        "OS Independent"
+    ],
+    "softwareRequirements": [
+        "python >= 3.6",
+        "astropy >= 4.0",
+        "colorama >= 0.3",
+        "h5py >= 2.8",
+        "lmfit >= 1.0.1",
+        "matplotlib >= 2.0",
+        "mpmath >= 1.0.0",
+        "numpy >= 1.18.1",
+        "scipy >= 1.2",
+        "setuptools >= 41.2"
+    ],
+    "relatedLink": [
+        "https://www.plasmapy.org/",
+        "https://docs.plasmapy.org/",
+        "https://app.element.io/#/room/#plasmapy-coredev:openastronomy.org",
+        "https://plasmapy.discourse.group/",
+        "https://groups.google.com/forum/#!forum/plasmapy",
+        "https://zenodo.org/communities/plasmapy",
+        "https://github.com/PlasmaPy/PlasmaPy-PLEPs",
+        ""
+    ],
+    "author": [
+        {
+          "@type": "Organization",
+          "name": "PlasmaPy Community"
+        },
+        {
+            "@type": "Person",
+            "@id": "https://orcid.org/0000-0001-6079-8307",
+            "givenName": "Erik",
+            "familyName": "Everson",
+            "email": "eeverson@ucla.edu",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "UCLA"
+            }
+        },
+        {
+            "@type": "Person",
+            "@id": "https://orcid.org/0000-0001-6291-8843",
+            "givenName": "Dominik",
+            "familyName": "Stańczak",
+            "email": "stanczakdominik@gmail.com",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "University of Warsaw"
```

```suggestion
                "name": "IPPLM"
```
let's just say I'm not feeling the affiliation with my university right now. :sweat_smile: